lib/kafka.ts

```diff
@@ -58,3 +58,75 @@ export const waitForKafkaReady = async function (page: Page, name: string) {
     timeout: config.kafkaInstanceCreationTimeout
   });
 };
+
+export const manageAccess = async function (page: Page, id: string, desc: string) {
+  await page.getByTestId('actionManagePermissions').click();
+
+  await page.getByRole('button', { name: 'Options menu' }).click();
+
+  await page.getByRole('option', { name: id + ' ' + desc }).click();
+
+  await page.getByRole('button', { name: 'Next' }).click();
+
+  await page.getByTestId('permissions-dropdown-toggle').click();
```

Comment:
Same here.

Reply:
What do you suggest in this case to locate the '>' dropdown @mmusil ? 

![image](https://user-images.githubusercontent.com/100594859/205649688-9c57c266-a612-440a-bc28-47280d8f9fbf.png)

Reply:
What about `aria-label="Select"`? :)

Reply:
Let me try

Reply:
Can I resolve this conversation? @mmusil